apps/backend/src/app.module.ts

```diff
@@ -34,23 +32,7 @@ import { TransactionModule } from './transaction/transaction.module';
     ReportingModule,
     NotificationModule,
     ConfigModule.forRoot({
-      ignoreEnvFile:
-        process.env.RUNTIME_ENV === 'local' || process.env.RUNTIME_ENV === 'ci'
-          ? false
-          : true,
-    }),
-    AwsSdkModule.forRoot({
-      defaultServiceOptions: {
-        ...(process.env.RUNTIME_ENV === 'local' ||
-        process.env.RUNTIME_ENV === 'ci'
-          ? {
-              endpoint: process.env.AWS_ENDPOINT || 'http://localhost:9000',
-              region: 'ca-central-1',
-              s3ForcePathStyle: true,
-            }
-          : {}),
-      },
-      services: [S3, SNS],
+      ignoreEnvFile: process.env.NODE_ENV === 'production',
```

Comment:
why do we no longer ignore the env file in dev/test?

Reply:
NODE_ENV==='production' applies to dev/test. The RUNTIME_ENV differentiates dev/test/prod

Reply:
Previously we were using the "nest-aws-sdk" (open source pkg) to set those defaults in the app module and thety were available in the services using the "@InjectAWS" decorator. That pkg was not compatible with v3. There is a similar pkg we could use instead: [here](https://www.npmjs.com/package/aws-sdk-v3-nest) but it didn't seem necessary....

Reply:
Perfect, I was hoping that was how we were using it but given how it was written before I wasn't entirely sure!